scanners/android/android.go

```diff
@@ -40,7 +40,18 @@ func (scanner *Scanner) DetectPlatform(searchDir string) (_ bool, err error) {
 		return false, fmt.Errorf("failed to search for build.gradle files, error: %s", err)
 	}
 
-	return len(scanner.ProjectRoots) > 0, err
+	isRecognized := len(scanner.ProjectRoots) > 0
```

Comment:
I think this is not necessary, can be moved to the if simply, also then the return can be something like 
`return len(scanner.ProjectRoots) > 0, err`

Reply:
It is not immediately obvious, that `len(scanner.ProjectRoots) > 0` means, that the scanner successfully recognized a project. One must read and interpret the surrounding code. Using the properly named variable, this is not an issue.